backend/src/main/java/com/redhat/cloud/notifications/db/repositories/BehaviorGroupRepository.java

```diff
@@ -37,6 +38,26 @@ public class BehaviorGroupRepository {
     @Inject
     FeatureFlipper featureFlipper;
 
+    public BehaviorGroup get(@NotNull UUID behaviorGroupId) {
+        return entityManager.find(BehaviorGroup.class, behaviorGroupId);
+    }
+
+    public BehaviorGroup createFull(String accountId, @Valid BehaviorGroup behaviorGroup, List<UUID> endpoints, Set<UUID> eventTypes) {
+        BehaviorGroup saved = create(accountId, behaviorGroup);
+        if (endpoints != null) {
+            updateBehaviorGroupActions(accountId, saved.getId(), endpoints);
+        }
+
+        if (eventTypes != null) {
+            updateBehaviorEventTypes(accountId, saved.getId(), eventTypes);
+        }
+
+        entityManager.flush();
+        entityManager.refresh(saved);
```

Comment:
I believe it was the cause - i'll do some cleanup and check.

Reply:
I get null pointer exceptions when trying to access the `actions`, by the time I'm setting the response.

I guess this is because the method to update event types and actions use native queries to do the update and by doing that hibernate does not get notified.